2023/political-donations-compare/main.js

```diff
@@ -0,0 +1,139 @@
+// For API and chart documentation please look here:
+
+const data_2020 = [
+    195, 247, 308, 487, 548, 655, 795, 872, 933, 933, 994, 994, 994, 994, 994,
+    1025, 1649, 1663, 1669, 1700, 1822, 1852, 1932, 1993, 2290, 2324, 2482,
+    2529, 2915, 3084, 3529, 3602, 3691, 3813, 4003, 4228, 4269, 4396, 4488,
+    4561, 9405, 10683, 10683, 10835, 10987, 11169, 11169, 11182, 11226, 12077,
+    12263, 12287, 14886, 14958, 15236, 15491, 15868, 16002, 16762, 16823, 16865,
+    16938, 17252, 17476, 17561, 17755, 17876, 17937, 17937, 18034, 18246, 19460,
+    19498, 20224, 20367, 20752, 20832, 22065, 24879, 24907, 24907, 25103, 25436,
+    29714, 30053, 30307, 30750, 32129, 32266, 34470, 34618, 34703, 34802, 35067,
+    35699, 35740, 36019, 36139, 36217, 36251, 36464, 36497, 36515, 36919, 37076,
+    37330, 37361, 37375, 37417, 37489, 38735, 38828, 38894, 38972, 39277, 39518,
+    40011, 40177, 40856, 41384, 41411, 41561, 41688, 42124, 42364, 42405, 42685,
+    42902, 43382, 43798, 43902, 44023, 44053, 44092, 44176, 44228, 44427, 44440,
+    44475, 44493, 44535, 44672, 44725, 44818, 44854, 45247, 45519, 45685, 45997,
+    47167, 48883, 48883, 48955, 49136, 49155, 49261, 49592, 49852, 50004, 50185,
+    50204, 51027, 51399, 52183, 52259, 52292, 52353, 53033, 53058, 53105, 53141,
+    53589, 53987, 54041, 54611, 54674, 55677, 58828, 60486, 61216, 63457, 63559,
+    63727, 63889, 63893, 63982, 64035, 64329, 64695, 65152, 65484, 65654, 65686,
+    65698, 65977, 66157, 66235, 66682, 67042, 68157, 68179, 68366, 70152, 71425,
+    71437, 71683, 71835, 72258, 72388, 72570, 73323, 73930, 74052, 74381, 74630,
+    74842, 75034, 76387, 76658, 77096, 77232, 77324, 77680, 78033, 78885, 78918,
+    79154, 79394, 79627, 79741, 79880, 80003, 80492, 82169, 82261, 82411, 82486,
+    82645, 83472, 83704, 84125, 84827, 86844, 86997, 87257, 87745, 87922, 88022,
+    88245, 88582, 90211, 90563, 90766, 91251, 93422, 96324, 98930, 99218, 99641,
+    99745, 99923, 100597, 100963, 101071, 101789, 102717, 102919, 103856,
+    104564, 105173, 105635, 106754, 115976, 116954, 117828, 118277, 118444,
+    118706, 118845, 119326, 120701, 120966, 121225, 124951, 125202, 125602,
+    126926, 127558, 127923, 128501, 128768, 128899, 129088, 130394, 131811,
+    132297, 132930, 133056, 133239, 133879, 134156, 134317, 135474, 136620
+];
+
+const data_2024 = [
```

Comment:
JavaScript uses camel case as a variable naming paradigm. This should be `data2024`. The same would apply to other variables throughout the JavaScript files.